var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -64,3 +65,11 @@ def cmake_args(self):
                 'ON' if self.run_tests else 'OFF')
         ]
         return args
+
+    def setup_environment(self, spack_env, run_env):
+        super(PyPybind11, self).setup_environment(spack_env, run_env)
+        spack_env.set('PYBIND11_USE_CMAKE', 1)
+
+    def install(self, spec, prefix):
+        super(PyPybind11, self).install(spec, prefix)
+        python('setup.py', 'install', '--prefix={0}'.format(prefix))
```

Comment:
Can you replace `python('setup.py', ...)` with `setup_py(...)`? This adds the `--no-user-cfg` flag used elsewhere in Spack. We also usually add `'--single-version-externally-managed', '--root=/'` for `setuptools` packages to prevent the installation of eggs.

Reply:
Check